cmd/it_test.go

```diff
@@ -17,6 +17,7 @@ func Test_getTunnelConfig(t *testing.T) {
 		{"Invalid(gibberish)", "kajsbdf&*(&", tunnelConf{}, true},
 		{"Invalid(wrong order)", "80:http", tunnelConf{"", ""}, true},
 		{"Invalid(Extra values)", "http:80:https", tunnelConf{}, true},
```

Comment:
"Couldn't parse [forward_type]:[local_port] - did you mean `http:80`" (with whatever they typed)